axes/handlers/cache.py

```diff
@@ -78,9 +79,10 @@ def user_login_failed(self, sender, credentials: dict, request=None, **kwargs):
             return
 
         username = get_client_username(request, credentials)
-        if settings.AXES_ONLY_USER_FAILURES and username is None:
+        lockout_parameters = list(get_lockout_parameters(request, credentials))
```

Comment:
Since AXES_LOCKOUT_PARAMETERS could be any iterable, for simplicity, I convert it to list